orbit-2-core/src/main/java/com/babylon/orbit2/BaseDslPlugin.kt

```diff
@@ -123,29 +121,40 @@ object BaseDslPlugin : OrbitDslPlugin {
         @Suppress("UNCHECKED_CAST")
         return when (operator) {
             is Transform<*, *, *> -> flow.map {
+                if (operator.registerIdling) containerContext.settings.idlingRegistry.increment()
```

Comment:
Seeing as this is used everywhere is it worth making this a block function instead?